patterns/gulpfile.js

```diff
@@ -1,264 +0,0 @@
-    .pipe( gulp.dest( cssDestination ) );
-}
-
-/**
- * Watchs any change on sassDirectories path and on the sassEntryFile. If any
- * change is detected the `styles` task is triggered.
- */
-function stylesWatch() {
-  log.success( 'Start to watch for .scss changes' );
-  watch( sassFiles, { ignoreInitial: true }, styles );
-}
-
-/**
- * Function that is used to lint on all the sass files and follow all the rules
- * specified on .sass-lint.json
- */
-function stylesLint() {
-  return gulp.src( sassFiles )
-    .pipe( sassLint() )
-    .pipe( sassLint.format() )
-    .pipe( sassLint.failOnError() );
-}
-
-/**
- * Function used to lint on the JS files, using the rules specified on
- * .eslintrc.json.
- */
-function jsLint() {
-  const jsFiles = [
-    './main.js',
-    './{' + appDirectories.join( ',' ) + '}/**/*.js',
-  ];
-  return gulp.src( jsFiles )
-    .pipe( eslint() )
-    .pipe( eslint.format() )
-    .pipe( eslint.failAfterError() );
-}
-
-const webpackConfig = require('./webpack.config.js');
-const jsEntryFile = './main.js';
-
-/**
- * Function used to create the JS to be used on specifc pages, by default
- * creates a source map associated with each JS file to make easier to debug.
- */
-function js() {
-  let options = Object.assign(webpackConfig, { devtool: 'source-map' });
-  return jsTask( options );
-}
-
-/**
- * Watches the changes of the JS and creates a single file with source maps,
- * the genreated JS is on dev mode.
- */
-function jsWatch() {
-  let options = Object.assign( webpackConfig, {
-    devtool: 'source-map',
-    watch: true,
-    module: {
-      loaders,
-    }
-  });
-  return jsTask( options );
-}
-
-/**
- * Function that creates a Build file, in this case it creates a minifed version
- * of the generated JS useful for production purposes.
- */
-function jsBuild() {
-  let plugins = [
-    new webpack.optimize.UglifyJsPlugin({
-      compress: {
-        warnings: false
-      }
-    })
-  ];
-  let options = Object.assign( webpackConfig, {
-    plugins,
-    module: {
-      loaders,
-    }
-  });
-  return jsTask( options );
-}
-
-/**
- * Function used to run webpack with a set of custom options based on the task
- * being executed.
- *
- * @param object options The options to be used to run webpack, by default uses webpack.config.js.
- * @return stream Returns a gulp task to be used.
- */
-function jsTask( options ) {
-  return gulp.src( jsEntryFile )
-    .pipe( webpackStream( options, null, function(err, stats) {
-      if ( stats.compilation.errors.length > 0 ) {
-        notify.onError({
-            title: "JS Error",
-            message: stats.compilation.errors[0].error
-        })(stats.compilation.errors[0].error);
-      }
-    }))
-    .pipe( gulp.dest( './static/js' ) );
-}
-
-/*
- * Function used to log data to the console via gutil function
- */
-const log = {
-  success(msg) {
-    gutil.log( gutil.colors.green( msg ) );
-  }
-}
-
-function icons() {
```

Comment:
as you removed the icons building then the use_icons() in the functions.php should be removed too

Reply:
deleted